src/hooks/useClientSideV3Trade.ts

```diff
@@ -34,15 +34,23 @@ export function useClientSideV3Trade<TTradeType extends TradeType>(
       : [otherCurrency, amountSpecified?.currency]
   const { routes, loading: routesLoading } = useAllV3Routes(currencyIn, currencyOut)
 
-  const quoter = useV3Quoter()
   const { chainId } = useWeb3React()
+  // Chains deployed using the deploy-v3 script only deploy QuoterV2.
+  const useQuoterV2 = useMemo(
+    () => Boolean(chainId && [SupportedChainId.CELO, SupportedChainId.CELO_ALFAJORES].includes(chainId)),
```

Comment:
`isCelo` helper? maybe even a `isQuoterV2Chain`?

Reply:
I believe this helper already exists in constants, and just needs to be exported.